src/pykeen/training/training_loop.py

```diff
@@ -115,13 +116,13 @@ def _get_optimizer_kwargs(optimizer: Optimizer) -> Mapping[str, Any]:
 
 
 def _get_lr_scheduler_kwargs(lr_scheduler: LRScheduler) -> Mapping[str, Any]:
-    lr_scheduler_kwargs = lr_scheduler.state_dict()
-    lr_scheduler_kwargs = {
+    # note: this seems to be a pretty unsafe method to derive __init__ kwargs...
```

Comment:
haha mood. This problem isn't generally solvable since anyone can write a class that has non-trivial init parameters that get modified/renamed in `__init__`.